src/helm/benchmark/metrics/basic_metrics.py

```diff
@@ -278,6 +278,22 @@ def tokenize(self, text: str) -> List[str]:
     return partial(rouge_score, scorer=scorer, rouge_type=rouge_type)
 
 
+def chinese_math_exact_match_indicator(gold: str, pred: str) -> float:
+    """
+    Exact match that only cares the last math expression.
+    Common math expressions are numbers and fractions.
+    """
+    pattern = r"[-+*/%\.\(\)\d]+"
+    matches = re.findall(pattern, pred)
+    if matches:
+        pred = matches[-1].lstrip(")")
+    for term in ["\n", "。", "，"]:
```

Comment:
You can remove the truncation here, as HELM will perform it in adaptation (via `stop_sequences`).

Reply:
Adjusted.